packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -1203,7 +1206,16 @@ export default class Meeting extends StatelessWebexPlugin {
 
       return Promise.resolve();
     } catch (err) {
-      if (err instanceof MeetingInfoV2PasswordError) {
+      if (err instanceof MeetingInfoV2PolicyError) {
+        this.meetingInfoFailureReason = MEETING_INFO_FAILURE_REASON.POLICY;
+        this.meetingInfoFailureCode = err.wbxAppApiCode;
+
+        if (err.meetingInfo) {
+          this.meetingInfo = err.meetingInfo;
```

Comment:
Normally when we get meeting info successfully, we also call `this.parseMeetingInfo()` which sets some things up on the Meeting object. I'm not sure, but maybe we should be doing this here too?

Reply:
I don't think we are parsing the meeting info for a captcha or password required meeting info. When a policy error happens the meeting is unjoinable. I don't think any of the things that are set up in parseMeetingInfo are useful in this case.

Reply:
ok, fair enough, the password error handling code is setting `this.meetingNumber` but I guess it's ok to not set anything for policy error